standard/expressions.md

```diff
@@ -1275,6 +1278,36 @@ In a member access of the form `E.I`, if `E` is a single identifier, and if the
 > ```
 > Within the `A` class, those occurrences of the Color identifier that reference the Color type are delimited by `**`, and those that reference the Color field are not. *end example*
 
+### §null-conditional-member-access Null Conditional Member Access
+
+A *null_conditional_member_access* consists of a *primary_expression* followed by a "`?.`" token, followed by an *Identifier*, optionally followed by a *type_argument_list*.
+
+```ANTLR
+null_conditional_member_access
+    : primary_expression '?.' Identifier type_argument_list?
+    ;
+```
+
+A  *null_conditional_member_access* expression `E` is of the form `P?.I`. Let `T` be the type of the *member_access*  expression `P.I`. The meaning of `E` is determined as follows:
+
+- If `T` is a type parameter that is not known to be a reference type or a non-nullable value type, a compile-time error occurs.
+- If `T` is a non-nullable value type, then the type of `E` is `T?`, and the meaning of `E` is the same as the meaning of:
+	> ```csharp
+	> ((object)P == null) ? (T?)null : P.I
+	> ```
+  Except that `P` is evaluated only once.
+- Otherwise the type of `E` is `T`, and the meaning of `E` is the same as the meaning of:
```

Comment:
Any reason the indentation here isn't the same as in the lines above?

Reply:
Do you mean indenting the code within the bullet? That seems correct to me, does the Standard not do this? Or are you referring to something else?

Reply:
I suspect the output may be fine in each case, but it's a bit odd to see the inconsistency. Compare lines 1295 and 1300... I can't see why they'd have different numbers of spaces before the `>`

Reply:
It was probably down to my tab setting be 3 and bullet indentation just being 2. I pushed them back 1 space, no changed in output